src/html/logo.html

```diff
@@ -28,7 +28,7 @@ <h2 class="h2 weight-strong">Icon</h2>
   <h2 class="h2 weight-strong">Lockup</h2>
   <div class="figure-wrapper">
     <figure class="grid-block figure vertical figure-logo-lockup">
-      <img class="img" src="images/logo/runnable-logo/RGB/runnable-logo-rgb.svg" width="100%" alt"Lockup on Purple">
+      <img class="img" src="images/logo/runnable-logo/RGB/runnable-logo-rgb.svg" width="100%" height="140px" alt"Lockup on Purple">
```

Comment:
Height and width attributes [can only be in pixels](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-width) so you shouldn't specify "px" here. For the same reason, `100%` isn't valid for the width; you should change it to a pixel value.